react/leanShipping.js

```diff
@@ -0,0 +1,379 @@
+import estimateCalculator from '@vtex/estimate-calculator'
+import sumBy from 'lodash/sumBy'
```

Comment:
i think you could import it as named imports in one single line

Reply:
and the render do this optimation for u

Reply:
I import like this because it imports only the file of the function, not the lib itself so it's smaller